wire-compiler/src/test/java/com/squareup/wire/schema/WireRunTest.kt

```diff
@@ -151,6 +152,264 @@ class WireRunTest {
         .doesNotContain("interface RouteGetUpdatedBlueClient : Service")
   }
 
+  @Test
+  fun protoOnly() {
+    writeBlueProto()
+    writeRedProto()
+    writeTriangleProto()
+
+    val wireRun = WireRun(
+        sourcePath = listOf(Location.get("colors/src/main/proto")),
+        protoPath = listOf(Location.get("polygons/src/main/proto")),
+        targets = listOf(ProtoTarget(outDirectory = "generated/proto"))
+    )
+    wireRun.execute(fs, logger)
+
+    assertThat(fs.find("generated")).containsExactly(
+        "generated/proto/squareup/colors/blue.proto",
+        "generated/proto/squareup/colors/red.proto")
+    assertThat(fs.get("generated/proto/squareup/colors/blue.proto"))
+        .contains("message Blue {")
+    assertThat(fs.get("generated/proto/squareup/colors/red.proto"))
+        .contains("message Red {")
+  }
+
+  @Test @Ignore("To be implemented")
```

Comment:
Nuke these tests and fix Target to specify the emit types only when they make sense

Reply:
nuked

Reply:
will refactor the target in followup